swan-lake/learn-the-platform/run-in-the-cloud/function-as-a-service/aws-lambda.md

```diff
@@ -1,193 +1,129 @@
+
+To deploy the function, execute the command, which you get in the CLI output logs after you [build the function](#build-the-function). For example, see the sample command below.
+
+**Info:** When you are deploying, make sure to replace the `$FUNCTION_NAME`, `$LAMBDA_ROLE_ARN`, and `$REGION_ID` placeholders with the corresponding values you obtained when [setting up the prerequisites](#set-up-the-prerequisites).
+
+```bash
+	Run the following command to deploy each Ballerina AWS Lambda function:
+	aws lambda create-function --function-name $FUNCTION_NAME --zip-file fileb:///Users/user1/Desktop/aws_lambda_deployment/target/bin/aws-ballerina-lambda-functions.zip --handler aws_lambda_deployment.$FUNCTION_NAME --runtime provided --role $LAMBDA_ROLE_ARN --layers arn:aws:lambda:$REGION_ID:134633749276:layer:ballerina-jre11:6 --memory-size 512 --timeout 10
+
+	Run the following command to re-deploy an updated Ballerina AWS Lambda function:
+	aws lambda update-function-code --function-name $FUNCTION_NAME --zip-file fileb://aws-ballerina-lambda-functions.zip
 ```
-Now, you can build and deploy the function as in the previous sample.
-To invoke this function, create a DynamoDB table.
-1. Go to <a href="https://console.aws.amazon.com/iamv2/home#/roles" target="_blank">**roles**</a>, and add `AWSLambdaDynamoDBExecutionRole` to the created role in the prerequisites.
-2. Go to the <a href="https://us-west-1.console.aws.amazon.com/dynamodbv2" target="_blank">DynamoDB</a>.
-3. Click **Create Table**, enter the table name, partition key, and create the table (If you already have a table created, you can skip this step).
-4. Click on the DynamoDB table, and then click the **Exports and streams** tab.
-5. Click **enable DynamoDB stream details**, and select the key attributes only for the event type.
-6. Once it's enabled, click **Create a trigger**, select the `notifyDynamoDB` from the dropdown, and create a trigger.
 
-Now, add an entry to the DynamoDB table to invoke the Lambda function. For this, go to <a href="https://us-west-1.console.aws.amazon.com/dynamodbv2" target="_blank">**Items**</a> in the DynamoDB, select the table, and click **Create item**. Once the item is entered into the table, go to the Lambda function, and check the logs via CloudWatch to see the object identifier in the logs.
+>**Info:**  For the supported parameters, go to the <a href="https://docs.aws.amazon.com/cli/latest/reference/lambda/create-function.html" target="_blank">`create-function` documentation</a>. You might need to change parameters such as the `MemorySize` and `Timeout` depending on your application and connection speed. 
+
+## Invoke the function
+
+The deployed AWS Lambda function can be tested by invoking it directly using the CLI. 
```

Comment:
If we are not executing the function we wrote above, here, this information is wrong. Since we cover invocation part in new BBEs, do we need this explicit section? We can just put a paragraph if we needed to mention saying that you can invoke the function based on the trigger.